packages/textfield/test/textfield.test.ts

```diff
@@ -880,7 +880,20 @@ describe('Textfield', () => {
             await elementUpdated(el);
 
             expect(negativeHelpText.variant).to.equal('negative');
-            await findDescribedNode(name, descriptionNegative);
+            // There's an issue in the way Firefox processes the a11y tree for
+            // elements with an `invalid` attribute/property. The following try/catch
+            // wrapping preps the code to pass in that context regardless and error
+            // when our tooling no longer runs into this error.
+            try {
+                await findDescribedNode(name, descriptionNegative);
+                if (/Firefox/.test(window.navigator.userAgent)) {
```

Comment:
nit: it might be nice to move this regex into https://github.com/adobe/spectrum-web-components/blob/main/tools/shared/src/platform.ts to help standardize consumption with the rest of the repo

Reply:
Good idea. I don't actually want people using something like this in production, but much better to make mistakes in one place than a bunch of them...fixed.